src/interfaces/IApolloCoordinatorV2.sol

```diff
@@ -3,7 +3,6 @@ pragma solidity 0.8.20;
 
 interface IApolloCoordinatorV2 {
     struct PriceFeedRequest {
```

Comment:
actually we already don't need this struct at in V2, forget to remove

Reply:
Removed